test/test_att.py

```diff
@@ -29,3 +28,12 @@ def test_att():
 
     res = eca(x)
     assert res.shape == x.shape
+
+    cbam = CBAM(x.shape[1])
+
+    res = cbam(x)
+    assert res.shape == x.shape
+    
+
+if __name__ == "__main__":
```

Comment:
No need to have this lines here. You should run the specific test with `python -m pytest ./test/test_att.py`